idp-service/src/main/resources/migrations/scorecard/checks.json

```
@@ -509,5 +509,130 @@
     "tags": ["Software quality", "Development maturity"],
     "defaultBehaviour": "FAIL",
     "failMessage": "Number of workflows is less than or equal to 3."
+  },
+  {
+    "id": "652e8e9116ff017907f1bc64",
+    "accountIdentifier": "__GLOBAL_ACCOUNT_ID__",
+    "identifier": "__HARNESS__864af47a-cb3b-7d98-ada7-9478ab7d6a5e",
+    "name": "Number of open critical alerts reported by dependabot is less than 3",
```

```suggestion
    "name": "Open Dependabot alerts is less than 3",
  
    
```
IMO the names should be shorter and descriptions have more details.

---

Also please use the correct case when writing a Product/Brand/Feature name. `Dependabot`.

---

Done